cost-analyzer/values.yaml

```
@@ -350,6 +369,61 @@ global:
 upgrade:
   toV2: false
 
+finops-agent:
+  ## Kubecost 2.9 is intended to facilitate the upgrade to Kubecost 3.0.
+  ## When upgrading to Kubecost 2.9, the finops-agent is enabled by default.
+  ## This can be disabled if you are not upgrading the cluster to Kubecost 3.0 or do not mind losing a partial day of data.
+  enabled: true
+  serviceAccount:
+    create: true
+    name: ""  ## recommended to use the same service account as .Values.serviceAccount.name
```

Is the idea here, that if the cost-analyzer ServiceAccount is configured with IRSA for s3 permissions. We want to let the finops-agent use the same ServiceAccount so it can also write to the bucket?

---

exactly, just trying ot simplify the changes needed.

you think there is a better method?

---

No, this makes sense to me! Good catch! What do you think about this updated comment?

```suggestion
    name: ""  # Set this to the same value as `.Values.serviceAccount.name` if you have attached an IAM role to your ServiceAccount.
```